.swift-format

```diff
@@ -33,7 +33,7 @@
         "AvoidRetroactiveConformances": true,
         "BeginDocumentationCommentWithOneLineSummary": false,
         "DoNotUseSemicolons": false,
-        "DontRepeatTypeInStaticProperties": true,
+        "DontRepeatTypeInStaticProperties": false,
```

Comment:
Strange that the [lint check fails](https://github.com/huggingface/swift-transformers/actions/runs/18075600846/job/51431586406?pr=274) even when this is disabled. FWIW, I also can't reproduce the lint failure locally...

> Sources/Models/LanguageModel.swift:132:20: warning: [DontRepeatTypeInStaticProperties] remove the suffix 'Keys' from the name of the variable 'presentKeys'